web/src/app/webview/src/release-main.ts

```diff
@@ -1,4 +1,15 @@
 import KeymanEngine from './keymanEngine.js'
 import { Worker } from '@keymanapp/lexical-model-layer/web'
 
-window['keyman'] = new KeymanEngine(Worker.constructInstance(), window.location.href);
\ No newline at end of file
+ /**
+  * Determine path and protocol of executing script, setting them as
+  * construction defaults.
+  *
+  * This can only be done during load when the active script will be the
+  * last script loaded.  Otherwise the script must be identified by name.
+ */
+ var scripts = document.getElementsByTagName('script');
+ var ss = scripts[scripts.length-1].src;
+ var sPath = ss.substr(0,ss.lastIndexOf('/')+1);
+
+window['keyman'] = new KeymanEngine(Worker.constructInstance(), sPath);
```

Comment:
Oh yeah... it might be wise to set the path configuration correctly _for release builds too_.  🤦 

Well, at least it was a super-easy fix once I found the problem.  Was config-related too... just in the _Web_ "config", not the mobile-app config.